index.html

```diff
@@ -21,26 +21,26 @@ <h2>Profile</h2>
     </section>
     <section class="work-experience">
       <h2>Work Experience</h2>
-      <h3><time datetime="2016-09-30">2016,</time> GMDP,Avaya Client Services</h3>
+      <h3><time datetime="2016-09-30">2016.09.30 <abbr title="Global Management Development Program">GMDP</abbr>,Avaya Client Services</h3>
       <ul>
         <li>Worked with various teams in Avaya Client Services to prioritize bugs, enhancements and new business processes.</li>
         <li>Established the SEE (Services, Entitlement Enablement) forum to communicate, plan and address key projects within ACS with stakeholders.</li>
       </ul>
-      <h3><time datetime="2014-09-12">2014,</time> Server, Bar Italia</h3>
+      <h3><time datetime="2014-09-12">2014.09.12</time> Server, Bar Italia</h3>
       <ul>
         <li>Worked with a small team of 8 people and provided exceptional customer service in a small boutique restaurant off of Madison Avenue.</li>
         <li>I took orders, cleared tables, cleaned silverware, made coffee and waited at private events.</li>
       </ul>
     </section>
     <section class="education">
       <h2>Education</h2>
-      <h3><time datetime="2017-01-12">2017,</time> Ubiqum Coding Academy</h3>
+      <h3><time datetime="2017-01-12">2017.01.12</time> Ubiqum Coding Academy</h3>
       <h4>Berlin,Germany</h4>
       <p>Five-month Intensive Full-stack Java development boot camp using a hand on approach to learning. Projects included making websites using HTML, CSS, JavaScript and bootstrap as well as using Java to make a simulation of the game battleship.</p>
-      <h3><time datetime="2016-09-12">2016,</time> Master of Arts / Conflict Management</h3>
+      <h3><time datetime="2016-09-12">2016.09.12</time> Master of Arts / Conflict Management</h3>
       <h4>Alice Salomon University, Berlin, Germany</h4>
       <p>Interdisciplinary Master's Program that focuses on social actions aimed at the participatory resolution of conflicts.</p>
-      <h3><time datetime="2015-09-12">2015,</time> Bachelor of Arts / Political Science</h3>
+      <h3><time datetime="2015-09-12">2015.09.12</time> Bachelor of Arts / Political Science</h3>
```

Comment:
<time datetime="2015-09-12">2015.09.12</time>
the date between tags, you need to use in the correct order. For example: 12.09.2015